packages/www/components/Plans/index.tsx

```diff
@@ -672,28 +547,54 @@ const Plans = ({ dashboard = false, stripeProductId }: PlanProps) => {
                 displayCheck={false}
                 title={
                   <Flex css={{ justifyContent: "center", width: "100%" }}>
-                    <span>Custom pricing</span>
+                    <span>Custom quote</span>
                   </Flex>
                 }
               />
               <Item
                 displayCheck={false}
                 title={
                   <Flex css={{ justifyContent: "center", width: "100%" }}>
-                    <span>Custom pricing</span>
+                    <span>Custom quote</span>
                   </Flex>
                 }
               />
               <Item
                 displayCheck={false}
                 title={
                   <Flex css={{ justifyContent: "center", width: "100%" }}>
-                    <span>Custom pricing</span>
+                    <span>Custom quote</span>
                   </Flex>
                 }
                 css={{ borderBottom: 0 }}
               />
-              <Item css={{ borderBottom: 0 }} displayCheck={false} title={""} />
+              <Item
+                displayCheck={false}
+                title={
+                  <Flex css={{ justifyContent: "center", width: "100%" }}>
+                    <span>Unlimited</span>
+                  </Flex>
+                }
+                css={{ borderBottom: 0 }}
+              />
+              <Item
+                displayCheck={false}
+                title={
+                  <Flex css={{ justifyContent: "center", width: "100%" }}>
+                    <span>$2.5k minimum spend</span>
```

Comment:
Same `*` explaining minimum spend?

Reply:
On the livepeer.studio pricing page the only * it's with annual commit, will ask hunter